src/schematools/types.py

```diff
@@ -714,6 +714,12 @@ def __init__(
 
     def __repr__(self):
         return f"<{self.__class__.__name__}: {self['id']}>"
+        prefix = ""
+        if self._parent_schema is not None:
+            prefix += f"{self._parent_schema.id}."
+        if self._parent_table is not None:
+            prefix += f"{self._parent_table.id}."
+        return f"<{self.__class__.__name__}: {prefix}{self['id']}>"
```

Comment:
You forgot to remove the original implementation, above.

Reply:
yikes. debugging mistakes..